cmd/collection_bench_matrix/README.md

```
@@ -14,9 +14,11 @@ OUT=/tmp/collection-matrix-$(date +%s)
 ./bin/collection-bench-matrix -out-dir "$OUT" -batch-size 16000 -benchtime 100000x
 ```
 
-By default, TreeDB collection cells use the `production_wal_on_fast` engine
-(WAL on with the fast/relaxed sync profile). Pass `-engine production_fast` when
-you need the no-WAL cached control.
+The current matrix engine names are legacy benchmark-harness labels. The default
+`production_wal_on_fast` engine is retained only until the harness migration in
+#2148. Current TreeDB server/profile guidance should use `command_wal_durable`,
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)